### PR TITLE
React Native "Find Meal" App

### PR DESCRIPTION
**Pull request by destinyleia**
My reflections on how this week's project turned out:
Not as I wanted my app to be, but close enough !

Things I'd like to have clarified or explained in more detail:
More about apis and native

### CODE REVIEW

File: components/MealList.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
+import React, { useState } from 'react'
+import { Vibration } from 'react-native'
+import styled from 'styled-components/native'
+
+import backgroundImage from '../assets/background-fruits.png'
+
+const MealsArray = [
```
Comment (EmOlsz) [inline components/MealList.js:7]: Capitalised names are reserved for components only, not arrays

File: components/MealList.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  }
+
+  return (
+    <Container>
+      <BackgroundImageContainer
+        source={backgroundImage}>
+        <ButtonContainer
+          onPress={() => {
+            setRandomMeals()
+            Vibration.vibrate()
```
Comment (EmOlsz) [inline components/MealList.js:79]: If we need to pass more than one line of code to event handler, then we should move whole logic to separate function

File: components/MealList.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+  )
+}
+
+const BackgroundImageContainer = styled.ImageBackground`
+  flex: 1;
+  justify-content: center;
+`
+
+const MealContainer = styled.View`
+`
```
Comment (EmOlsz) [inline components/MealList.js:100]: I don't think you need this styled component

File: components/MealList.js [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+
+const Container = styled.View`
+flex: 1;
+`
+
+const ButtonContainer = styled.TouchableOpacity`
+top:  100px;
+margin: 100px;
```
Comment (EmOlsz) [inline components/MealList.js:118]: Wrong indentations

**Review by EmOlsz — CHANGES_REQUESTED** [review]

File: components/MealList.js (hunk 2 above)
Comment (destinyleia) [reply components/MealList.js:79]: I think I changed it right now - please check line 69 + 70 and 78 in the component MealList :)

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**